build.gradle

```diff
@@ -12,14 +12,17 @@ plugins {
 
 
 repositories {
-    mavenLocal()
+    // mavenLocal()
     maven {
         url = uri('https://repo.maven.apache.org/maven2')
     }
 
     // maven {
     //     url = uri('https://modusbox.jfrog.io/modusbox/libs-release')
     // }
+    maven {
+        url = uri('https://fynarfin.jfrog.io/artifactory/fyn-libs-snapshot-local/')
+    }
```

Comment:
Please rebase. This piece of code is not required as you are not pulling connector common from jfrog but rather pushing connector common to that jfrog URL through the gradle artifactory task in line 62.